pages/_document.tsx

```diff
@@ -12,6 +12,10 @@ export default class MyDocument extends Document {
       <Html>
         <Head>
           <HashiHead />
+          <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap"
+          />
```

Comment:
I got a warning about this not being in `_document`, so shifted it here.

Reply:
Can be handled outside of this PR, but not even sure Open Sans is being used in some fashion anymore.

Reply:
Even better!